iot/mqttsnippets/subscribe_gateway_device_topic.go

```
@@ -0,0 +1,98 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package mqttsnippets
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/eclipse/paho.mqtt.golang"
+)
+
+// [START iot_subuscribe_config_bound_device]
+
+// subscribeGatewayToDeviceTopic creates a gateway client that subscribes to a topic of a bound device.
+// Currently supported topics include: "config", "state", "commands", "errors"
+func subscribeGatewayToDeviceTopic(w io.Writer, projectID string, region string, registryID string, gatewayID string, deviceID string, privateKeyPath string, algorithm string, clientDuration int, topic string) {
```

Should return an error.